content/docs/migration_1.0/Python.md

```diff
@@ -37,50 +37,68 @@ with zenoh.open(zenoh.Config()) as session:
     session.declare_subscriber("my/keyepxr", lambda s: print(s))
     sleep(10) # subscriber stays in background and its callback can be called
     # `session.close()` will be called at the end of the block, and it will undeclare the subscriber
-```¬
+```
 
-## ZBytes, encoding, and (de)serialization
+## Value is gone, long live ZBytes
 
-### Encoding
+`Value` has been split into `ZBytes` and `Encoding`. `put` and other operations now requires a `ZBytes` payload, and builders accept an optional `Encoding` parameter. 
 
-`zenoh.Value` has been split into `zenoh.ZBytes` and `zenoh.Encoding`. Put and other operations now require a `ZBytes` payload, and accept an optional `Encoding`; the encoding is no longer automatically deduced from the payload type.
+`ZBytes` is a raw bytes container. It can be created directly from raw bytes/strings using `ZBytes` constructor. Then bytes can be retrieved using `ZBytes.to_bytes` or `ZBytes.to_string`. Sample payload is now a `ZBytes` instead of `bytes`.
 
-```python
-session.put("my/keyexpr", 42) # default encoding `zenoh/bytes`session.put("my/keyexpr", 42, encoding=zenoh.Encoding.ZENOH_INT64)
+- Zenoh 0.11.x
+
+```rust
+sample = subscriber.recv()
+my_string = sample.payload.decode("utf-8")
 ```
 
-Publishers can be declared with a default encoding, which will be used for each put operation.
+- Zenoh 1.0.0
+
+```rust
+sample = subscriber.recv()
+my_string = sample.payload.to_string()
+```
+
+You can look at a full set of examples in `examples/z_bytes.py`.
+
+### Serialization
```

Comment:
Same issue as on Rust: this is the only "sub-title", which makes the summary of the page look weird. Consider changing it.

```suggestion
## Serialization
```